Competition2017/src/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -49,7 +55,8 @@ public DriveSubsystem(WPIFactory factory, XPropertyManager propManager) {
         leftDriveEncoderTicksProp = propManager.createEphemeralProperty("Left drive encoder ticks", 0);
         rightDriveEncoderTicksProp = propManager.createEphemeralProperty("Right drive encoder ticks", 0);
         
-        ticksPerInch = propManager.createPersistentProperty("Ticks per inch", 25.33);
+        highGearTicksPerInch = propManager.createPersistentProperty("Ticks per inch for high gear", 48.8);
+        lowGearTicksPerInch = propManager.createPersistentProperty("Ticks per inch for low gear", 48.8);
```

Comment:
might be handy to start these at 2 different numbers by default so it'll be more evident they shouldn't match?